src/utils/TimeSlots.js

```diff
@@ -28,7 +28,7 @@ export function getSlotMetrics({ min: start, max: end, step, timeslots }) {
   for (let grp = 0; grp < numGroups; grp++) {
     groups[grp] = new Array(timeslots)
 
-    for (let slot = 0; slot < timeslots; slot++) {
+    for (let slot = 0; slot <= timeslots; slot++) {
```

Comment:
This will add an extra timeslot to every step, which probably isn't what you want.

Reply:
Ah yes, I see that now :) I didn't really dive into what all that code did, only tried it out and found it working. What do you suggest doing to give us that last slot? Only add an extra timeslot to the last group? As far as I could see it, adding an extra group would skewer the whole day a bit.

Reply:
Sorry, I don't know and I don't have time to dive into it right now.

Reply:
Or we could push an extra slot into the `slots` array, that way the last group won't have an extra slot, and it won't affect the rendering of the group. And then `closestSlotToPosition` would at least return the correct slot instead of `undefined` :)

Reply:
Oh sorry, didn't see the previous comment you made. But as far as I can see it, it works with just pushing an extra timeslot at the end of the slots array. I can push those changes, and you can review whenever you have time :)